### PR TITLE
Create DA solution

### PR DESCRIPTION
**Pull request by ocofaigh**
### Description

Create DA solution

### Release required?

- [ ] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

**Review by bhakta-ibm — COMMENTED** [review]
Two minor comments. Please take a look ans I can approve.

File: ibm_catalog.json [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+                {
+                  "profile_name": "IBM Cloud Framework for Financial Services",
+                  "profile_version": "1.5.0"
+                }
+              ]
+            },
+            "iam_permissions": [
+              {
+                "role_crns": [
+                  "crn:v1:bluemix:public:iam::::serviceRole:WriterManager"
```
[the remark this answers is not in the document]
Comment (ocofaigh) [reply ibm_catalog.json:57]: ah yes ok - I copied from the json file you sent me :)

**Review by bhakta-ibm — APPROVED** [review]
lgtm

**Review by MatthewLemmond — APPROVED** [review]

**Pull request merged**